docs/usage/configuration.md

```diff
@@ -227,29 +227,15 @@ Note that the `show-error-end` flag is always passed into mypy, so it does not n
 ## Overriding error messages
 
 All error messages from either pylint or PythonTA checkers can be overridden.
-To override the message for a specific check, you need to know the _checker class_ that checks for the message and the _module_ in which the class is defined.
-For example, to customize `E0111` (`bad-reversed-sequence`), we require knowing that this check is made in the `BasicChecker` in the `pylint.checkers.base` module.
+Custom messages are saved in a [TOML file](https://toml.io/en/), and any section headers within the TOML file will be ignored.
 
-Given this information, custom messages are saved in a [TOML file](https://toml.io/en/).
-Custom messages must be nested in the sequence _module_ -> _checker class_ -> _error code_.
-For example, we can override the error message for `E0111` in any of the following ways (taking advantage of TOML's flexibility with [nested keys](https://toml.io/en/v1.0.0#table)):
+For example, we can override the error message for `E0111` as follows:
 
 ```toml
-# Version 1
-["pylint.checkers.base".BasicChecker]
 E0111 = "reversed() can only be called on instances of sequence types like str, list, or tuple."
-
-# Version 2
-["pylint.checkers.base"]
-BasicChecker.E0111 = "reversed() can only be called on instances of sequence types like str, list, or tuple."
-
-# Version 3
-"pylint.checkers.base".BasicChecker.E0111 = "reversed() can only be called on instances of sequence types like str, list, or tuple."
 ```
 
 _Notes_:
 
-1. The module name (e.g., `pylint.checkers.base`) must be enclosed in quotes.
-2. For pylint messages, the [pylint documentation](https://pylint.readthedocs.io/en/latest/user_guide/messages/messages_overview.html) is helpful for determining where each check is defined.
-3. View the [default PythonTA message configuration](https://github.com/pyta-uoft/pyta/blob/master/python_ta/config/messages_config.toml) for examples of overriding messages.
-4. Custom message must use the same string conversion specifiers (e.g., `%s`) as the original message. See the pylint documentation for the original message format.
+1. View the [default PythonTA message configuration](https://github.com/pyta-uoft/pyta/blob/master/python_ta/config/messages_config.toml) for examples of overriding messages.
```

Comment:
Oh, please update the default PyTA `messages_config.toml` as well to remove all section headers.